src/index.js

```diff
@@ -86,6 +90,10 @@ export default class ScrollContainer extends Component {
   };
 
   onMouseMove = (e) => {
+    // If dragging within an 'ignored' element, prevent typical scrolling behaviour
```

Comment:
It's a bad idea to handle the drag preventing here, it will produce several bugs and even performance problems.

First of all, remember that drag scrolling will start when you will drag something at your ignored element, and it will only wait a moment when your mouse will leave ignored element.

Secondly, `e.target.closest` will check all parents of elements include the parents of `ScrollContainer`. It may produce undesirable behavior.

Test the following [solution](https://gist.github.com/Norserium/6fb9ecd2a70c885089b93e9018338256). Same idea, but different implementation.

Reply:
Works perfectly!